examples/create_postcards_from_csv/index.js

```diff
@@ -42,6 +42,11 @@ parse(input, function (err, rows) {
         name: row[0],
         car: row[3],
         mileage: row[4]
+      },
+      metadata: {
+        name: row[0],
+        customer_id: 'customer_123',
```

Comment:
Potential nit is this wouldn't **quite** fit the customer flow since the customer ID would presumably be coming from the CSV.

I would be down to either keep, add an ID field to the CSV, or remove it and replace `campaign` with a `campaign_id` if you're hoping to get an `id` field in the example.